### ISSUE TITLE
douyutv not working again?

### ISSUE THREAD

**Issue by DragonZ** [chrippa/livestreamer#1268]
- Input: livestreamer douyutv.com/208114 best
- Output: [cli][info] Found matching plugin douyutv for URL douyutv.com/208114
  error: Unable to validate JSON: Unable to validate key 'data': u'\u63a5\u53e3\u5df2\u5173\u95ed' does not equal None or Type of u'\u63a5\u53e3\u5df2\u5173\u95ed' should be 'dict' but is 'unicode'
- Used Version: livestreamer 1.12.2

I noticed that there was one similar issue a while ago. It seems that it happened again. Any idea about it?

**Comment by intact** [comment]
Your livestreamer is too old. You can use git version (or [Nightly build](http://docs.livestreamer.io/install.html#nightly-build) if you use Windows).

**Comment by DragonZ** [comment]
@intact You are right. Building from github source solves this issue. Will there be a new release soon?
